Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/misc.yml

```diff
@@ -66,6 +106,52 @@
     tags:
     - Fruit
 
+- type: entity
+  name: chocolate muffin
+  parent: FoodBakedMuffin
+  id: FoodBakedMuffinChocolate
+  description: A delicious and spongy chocolate muffin.
+  components:
+  - type: Sprite
+    state: muffin-chocolate
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 12
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 6
+        - ReagentId: Vitamin
+          Quantity: 2
+        - ReagentId: CocoaPowder
+          Quantity: 2
+
+- type: entity
+  name: banana muffin
+  parent: FoodBakedMuffin
+  id: FoodBakedMuffinBanana
+  description: A delicious and spongy banana muffin.
+  components:
+  - type: Sprite
+    state: muffin-banana
+  - type: SolutionContainerManager
+    solutions:
+      food:
+        maxVol: 12
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 6
+        - ReagentId: Vitamin
+          Quantity: 2
+        - ReagentId: JuiceBanana
+          Quantity: 2
+  - type: Tag
+    tags:
+    - Fruit
+    - Banana
```

Comment:
```suggestion
```
Remove the banana tag. That one is used for a cargo bounty and we don't want the muffins to fullfill it.